cmd/main.go

```diff
@@ -60,6 +60,7 @@ var (
 	enableLeaderElection = flag.Bool("leader-elect", false, "Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
 	leaderElectionID     = flag.String("leader-elect-id", "", "Lease name for leader election.")
 	scaleDownInterval    = flag.Duration("scale-down-interval", 55*time.Minute, "The scale down interval.")
+	scaleUpInterval      = flag.Duration("scale-up-interval", 0, "The scale up interval.")
```

Comment:
```suggestion
	scaleUpInterval      = flag.Duration("scale-up-interval", 60*time.Second, "The scale up interval.")
```

Set the same default value for CLI flag too, since this is the fallback value if `spec.ScaleConfig.ScaleupInterval` is not set.